compiler/Makefile

```diff
@@ -19,6 +19,7 @@ DATAMODEL_PATH ?= datamodel
 CONFIG_FILE ?= ""
 GENERATED_OUTPUT_DIRECTORY ?= generated
 COMPILER_SRC_DIRECTORY ?= ""
+FORCE ?= false
```

Comment:
@pavithra-kavach should we consider renaming this to FORCE_UPGRADE ?
We can take it up in a followup jira.

Reply:
Sure. I'll open a JIRA to review this.  Originally written as Force Upgrade, it was altered to Force after review suggestions.

Reply:
https://jira.eng.vmware.com/browse/NPT-803